docs/concepts/filters.md

```diff
@@ -0,0 +1,28 @@
+### Filters
```

Comment:
👉🏽👉🏽 Don't forget to add to each new docs markdown page a `title` and `weight`. 

The `title` will influence the left-hand navigation in our docs. The `weight` will enable you to order these pages as you need them once we determine a set content outline. 😃 

```
---
title: 'Overview'
weight: 0
---
```

Reply:
okay got it!👍